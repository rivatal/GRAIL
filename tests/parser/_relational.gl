1==1;
1!=1;
1<1;
1<=1;
1>1;
1>=1;
true and true;
true or false;