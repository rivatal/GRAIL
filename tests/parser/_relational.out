Expr(Binop(Num_Lit(1), Equal, Num_Lit(1)));
Expr(Binop(Num_Lit(1), Neq, Num_Lit(1)));
Expr(Binop(Num_Lit(1), Less, Num_Lit(1)));
Expr(Binop(Num_Lit(1), Leq, Num_Lit(1)));
Expr(Binop(Num_Lit(1), Greater, Num_Lit(1)));
Expr(Binop(Num_Lit(1), Geq, Num_Lit(1)));
Expr(Binop(Bool_lit(true), And, Bool_lit(true)));
Expr(Binop(Bool_lit(true), Or, Bool_lit(false)));
